modules/RTC/LocalSdpMunger.js

```diff
@@ -2,6 +2,7 @@
 
 import { getLogger } from 'jitsi-meet-logger';
 import * as MediaType from '../../service/RTC/MediaType';
+import PrefixedLogger from '../util/PrefixedLogger';
```

Comment:
I wonder if there is any value in putting this in jitsi-meet-logger ?

Reply:
Yes, maybe that would work better. Not sure when I get to that though...

Reply:
But do you mean just moving this class or integrating this somehow into the logger ?

I've noticed that the log method name now always comes from PrefixedLogger. Having that integrated would probably allow to overcome this limitation.

Reply:
I meant moving the class. Maybe a Proxy object could help? The bonus nachos is that you really only need to override a single method (log) in that case, right?